go/vt/sqlparser/testdata/select_cases.txt

```diff
@@ -15397,8 +15397,8 @@ END
 INPUT
 select t1.time+0,t1.date+0,t1.timestamp+0,concat(date," ",time), t1.quarter+t1.week, t1.year+timestampadd, timestampdiff from t1;
 END
-ERROR
-syntax error at position 107
+OUTPUT
+select t1.`time` + 0, t1.`date` + 0, t1.`timestamp` + 0, concat(`date`, ' ', `time`), t1.`quarter` + t1.`week`, t1.`year` + `timestampadd`, `timestampdiff` from t1
```

Comment:
This also parses on MySQL so we actually are better supporting things with these changes here.